core/tests/performance_framework/perf_services.py

```diff
@@ -240,8 +240,11 @@ def _setup_driver(self, proxy=None, use_proxy=False):
             chrome_options = webdriver.ChromeOptions()
             # Disable several subsystems which run network requests in the
             # background. This helps reduce noise when measuring network
-            # performance.
+            # performance. Also, disable prerendering by chrome which renders
+            # a page in the background leading to wrong test results.
```

Comment:
Hmm, the problem with testing with prerendering disabled is it leads to some skew. Not all browsers support it, but Chrome is being specifically used for the performance tests and Chrome supports prerendering. Users using Chrome will experience prerendering, so I think having the performance tests have parity with that is good.

That being said, I disagree with your sentiment that prerendering will have no effect on these performance tests if enabled. Won't it negatively impact the performance tests since other pages are not being loaded? That could cause skew in the other direction. If turning on prerendering does not impact the performance tests, then I think it should be enabled. If it does negatively impact them, then it should be disabled.

Either way, there's skew with what happens in the testing env versus production. We should decide which skew we can live with.